build/android/config.gni

```diff
@@ -5,4 +5,10 @@ declare_args() {
 }
 
 brave_android_manifest_includes = [ "//brave/android/java/AndroidManifest.xml" ]
-brave_chrome_java_deps = ["//brave/third_party/android_deps:google_play_services_safetynet_java"]
+
+brave_chrome_java_deps = [
+  "//brave/third_party/android_deps:google_play_services_safetynet_java",
+  "//brave/third_party/zxing:zxing_java",
```

Comment:
what is this stuff and is any of it actually relevant to the UI or was this stuff for the sync backend?

Reply:
I see, it's for QR code reading?

Reply:
Yes, it's the QR code libraries.